tests/e2e/default/transitions/forms/index.js

```diff
@@ -0,0 +1,99 @@
+const fs = require('fs');
+const path = require('path');
+const utils = require('@utils');
+
+const getFormDoc = (formId) => {
+  const xmlPath = path.join(__dirname, `./${formId}.xml`);
+  const xml = fs.readFileSync(xmlPath, 'utf8');
+  return {
+    _id: `form:${formId}`,
+    internalId: formId,
+    title: formId,
+    type: 'form',
+    _attachments: {
+      xml: {
+        content_type: 'application/octet-stream',
+        data: Buffer.from(xml).toString('base64'),
+      },
+    },
+  };
+};
+
+const getMutePersonForm = () => ({
+  ...getFormDoc('mute_person'),
+  context: {
+    person: true,
+    place: false,
+  },
+});
+
+const getUnmutePersonForm = () => ({
+  ...getFormDoc('unmute_person'),
+  context: {
+    person: true,
+    place: false,
+  },
+});
+
+const getMuteClinicForm = () => ({
+  ...getFormDoc('mute_clinic'),
+  context: {
+    person: false,
+    place: true,
+  },
+});
+
+const getUnmuteClinicForm = () => ({
+  ...getFormDoc('unmute_clinic'),
+  context: {
+    person: false,
+    place: true,
+  },
+});
+
+const getMuteNewClinicForm = () => ({
+  ...getFormDoc('mute_new_clinic'),
+  context: {
+    person: false,
+    place: true,
+  },
+});
+
+
+module.exports.uploadForms = async () => {
```

Comment:
```suggestion
const uploadForms = async () => {
```

Please update this line to be `const` and at the end export them as we are doing in most of our files.

```javascript
module.exports = {
  uploadForms,
};
```